eZ/Publish/Core/REST/Server/Controller/Role.php

```diff
@@ -115,6 +126,32 @@ public function createRole(Request $request)
     }
 
     /**
+     * Creates a new RoleDraft for an existing Role.
+     *
+     * @since 6.1
```

Comment:
@bdunogier Fixed, thanks. Set to 6.2 for now. Also added same for other new methods.